HashTable/HashTable/list.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <stdbool.h>
+
+// List
+typedef struct List List;
+
+// Create list
+List* createList(void);
+
+// Check emptiness of list
+// return true if list is empty;
+bool listIsEmpty(List* list);
+
+// Check availability of value in the list
+// return true if value exist
+bool listContains(List* const list, const char* const value);
+
+// Add value in list;
+// if the value is not in the list, add a new one
+// else increase the counter of frequency;
+// return -1 if out of memory error occured; 1 if the new value in the list appeared else 0
+int addValueToList(List* const list, const char* const value, const int frequency);
+
+// Return length of list
+int getLength(List* list);
```

Comment:
```suggestion
int getLength(const List* list);
```
И на все методы, которые не изменяют список, тоже такое сделать